qcoro/qml/qcoroqml.h

```diff
@@ -0,0 +1,13 @@
+// SPDX-FileCopyrightText: 2022 Jonah Brüchert <jbb@kaidan.im>
+//
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "qcoroqml_export.h"
+
```

Comment:
Please add `#Include "qcoroqmltask.h"` here. It's common for the "module headers" (like "qcorocore.h", "qcoronetwork.h", etc.) to include all other files in the module (similar to including "QtCore" includes everything in the QtCore module).